integrations/gitlab/gitlab_integration/events/setup.py

```diff
@@ -30,6 +31,10 @@ class GitlabEventListenerConflict(OceanAbortException):
     pass
 
 
+class GitlabIllegalEventName(OceanAbortException):
```

Comment:
I feel like we can already dedicate specific errors file for all the gitlab custom errors

Reply:
done